components/aws/sagemaker/batch_transform/component.yaml

```diff
@@ -71,7 +71,7 @@ outputs:
   - {name: output_location,    description: 'S3 URI of the transform job results.'}
 implementation:
   container:
-    image: carowang/kubeflow-pipeline-aws-sm:20190801-01
+    image: carowang/kubeflow-pipeline-aws-sm:latest
```

Comment:
Using `latest` can cause existing pipelines to break once an incompatible image version is pushed (e.g. with some command-line parameter deprecated). Pinning the image version is preferable.

Reply:
Ah makes sense. Will revert back to using the image version.